### PR TITLE
Dịch "mlp-gluon_vn"

### LINKED ISSUE #839: Dịch "mlp-gluon_vn"
Phần cần dịch: https://github.com/aivivn/d2l-vn/blob/master/chapter_multilayer-perceptrons/mlp-gluon_vn.md

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by lyphilong**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #839 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_multilayer-perceptrons/mlp-gluon_vn.md [hunk 1]
```diff
@@ -5,14 +5,14 @@
 # Concise Implementation of Multilayer Perceptron
 -->
 
-# *dịch tiêu đề phía trên*
+# Triển khai súc tích của Mạng nơron Perceptron Nhiều tầng
```
Comment (tiepvupsu) [inline chapter_multilayer-perceptrons/mlp-gluon_vn.md]:
```suggestion
# Triển khai súc tích của Mạng nơron Perceptron Đa tầng
```

**Comment by lyphilong** [comment]
@tiepvupsu dạ em cảm ơn anh ạ.

File: chapter_multilayer-perceptrons/mlp-gluon_vn.md [hunk 2]
```diff
@@ -76,7 +79,9 @@ d2l.train_ch3(net, train_iter, test_iter, loss, num_epochs, trainer)
 3. Try different schemes for initializing the weights. What method works best?
 -->
 
-*dịch đoạn phía trên*
+1. Bằng việc thử thêm số lượng khác nhau của tầng ẩn, bạn hãy xem cài đặt nào cho kết quả tốt nhất (giữ nguyên giá trị các tham số khác và siêu tham số)?
```
Comment (tiepvupsu) [inline chapter_multilayer-perceptrons/mlp-gluon_vn.md]:
```suggestion
1. Bằng việc thử thêm số lượng khác nhau của tầng ẩn, bạn hãy xem thiết lập nào cho kết quả tốt nhất (giữ nguyên giá trị các tham số khác và siêu tham số)?
```

File: chapter_multilayer-perceptrons/mlp-gluon_vn.md [hunk 3]
```diff
@@ -47,14 +49,15 @@ net.initialize(init.Normal(sigma=0.01))
 Note that Gluon, as usual, automatically infers the missing input dimensions to each layer.
 -->
 
-*dịch đoạn phía trên*
+Lưu ý rằng, như thường lệ, Gluon sẽ tự động đưa vào số chiều đầu vào bị thiếu cho mỗi tầng.
```
Comment (rootonchair) [inline chapter_multilayer-perceptrons/mlp-gluon_vn.md]:
```suggestion
Lưu ý rằng, như thường lệ, Gluon sẽ tự động suy ra số chiều đầu vào bị thiếu cho mỗi tầng.
```

File: chapter_multilayer-perceptrons/mlp-gluon_vn.md [hunk 4]
```diff
@@ -47,14 +49,15 @@ net.initialize(init.Normal(sigma=0.01))
...
 
 <!--
 The training loop is *exactly* the same as when we implemented softmax regression.
 This modularity enables us to separate matterns concerning the model architecture from orthogonal considerations.
 -->
 
-*dịch đoạn phía trên*
+Quá trình lặp để huấn luyện giống *chính xác* với những gì khi ta lập trình hồi quy softmax.
+Lập trình mô-đun này cho phép ta tách các mô hình riêng biệt liên quan đến kiến trúc của mô hình ra để xem xét trực tiếp.
```
Comment (rootonchair) [inline chapter_multilayer-perceptrons/mlp-gluon_vn.md]:
```suggestion
Lập trình hướng mô-đun như vầy cho phép ta tách các chi tiết liên quan đến kiến trúc của mô hình ra khỏi các mối bận tâm chính.
```
`orthogonal consideration` ở đây có nghĩa là các mối bận tâm không dính dáng gì đến kiến trúc mô hình, hay còn gọi là độc lập. Giống như khi hai vector trực giao với nhau thì chúng không phụ thuộc vào nhau vậy.

Comment (lyphilong) [reply chapter_multilayer-perceptrons/mlp-gluon_vn.md]: @rootonchair em cảm ơn anh ạ

File: chapter_multilayer-perceptrons/mlp-gluon_vn.md [hunk 5]
```diff
@@ -5,14 +5,14 @@
 # Concise Implementation of Multilayer Perceptron
 -->
 
-# *dịch tiêu đề phía trên*
+# Triển khai súc tích của Mạng Perceptron Đa tầng
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/mlp-gluon_vn.md]:
```suggestion
# Triển khai súc tích của Perceptron Đa tầng
```

File: chapter_multilayer-perceptrons/mlp-gluon_vn.md [hunk 6]
```diff
@@ -34,7 +34,9 @@ The first is our hidden layer, which contains *256* hidden units and applies the
 The second, is our output layer.
 -->
 
-*dịch đoạn phía trên*
+So với việc dùng gluon để lập trình hồi quy softmax (:numref:`sec_softmax_gluon`), khác biệt duy nhất ở đây là ta thêm *hai* tầng `Dense` (kết nối đầy đủ). (trong khi trước đây, ta chỉ cần thêm *một*).
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/mlp-gluon_vn.md]:
```suggestion
So với việc dùng gluon để lập trình hồi quy softmax (:numref:`sec_softmax_gluon`), khác biệt duy nhất ở đây là ta thêm *hai* tầng `Dense` (kết nối đầy đủ -- trong khi trước đây, ta chỉ cần thêm *một*). 
```

File: chapter_multilayer-perceptrons/mlp-gluon_vn.md [hunk 7]
```diff
@@ -47,14 +49,15 @@ net.initialize(init.Normal(sigma=0.01))
...
+Lưu ý rằng, như thường lệ, Gluon sẽ tự động suy ra số chiều đầu vào bị thiếu cho mỗi tầng.
 
 <!--
 The training loop is *exactly* the same as when we implemented softmax regression.
 This modularity enables us to separate matterns concerning the model architecture from orthogonal considerations.
 -->
 
-*dịch đoạn phía trên*
+Quá trình lặp để huấn luyện giống *chính xác* với những gì khi ta lập trình hồi quy softmax.
+Lập trình hướng mô-đun như vầy cho phép ta tách các chi tiết liên quan đến kiến trúc của mô hình ra khỏi các mối bận tâm chính.
```
Comment (duythanhvn) [inline chapter_multilayer-perceptrons/mlp-gluon_vn.md]:
```suggestion
Lập trình hướng mô-đun như vậy cho phép ta tách các chi tiết liên quan đến kiến trúc của mô hình ra khỏi các mối bận tâm chính.
```
Từ `vầy` nghe cutes ghê.

File: chapter_multilayer-perceptrons/mlp-gluon_vn.md (hunk 6 above)
Comment (tiepvupsu) [inline chapter_multilayer-perceptrons/mlp-gluon_vn.md]:
```suggestion
So với việc dùng gluon để lập trình hồi quy softmax (:numref:`sec_softmax_gluon`), khác biệt duy nhất ở đây là ta thêm *hai* tầng `Dense` (kết nối đầy đủ) trong khi trước đây, ta chỉ cần thêm *một*. 
```

Comment (lyphilong) [reply chapter_multilayer-perceptrons/mlp-gluon_vn.md]: @duythanhvn Phần này em apply của anh @tiepvupsu  nha anh, tại em thấy cách viết của ảnh ổn hơn ạ.

**Comment by lyphilong** [comment]
@rootonchair mình đã bổ sung thêm vào rồi nha
